docs/data/routes/docs/client-frameworks/angular/angular-tips/en.md

```diff
@@ -120,8 +120,40 @@ There aren't changes in your component module. If you use multiple components la
     JssModule.withComponents([
       // non-lazy components
     ], [
-      { path: 'FirstComponent', loadChildren: './my/my.module#MyModule'},
-      { path: 'SecondComponent', loadChildren: './my/my.module#MyModule'},
+      { path: 'FirstComponent', loadChildren: import('./my/my.module').then(mod => mod.MyModule)},
+      { path: 'SecondComponent', loadChildren: import('./my/my.module').then(mod => mod.MyModule)},
+    ]),
+  ],
+  exports: [
+    JssModule,
+    AppComponentsSharedModule,
+  ],
+  declarations: [
+    // non-lazy components
+  ],
+})
+export class AppComponentsModule { }
+```
+
+Components can also have a  for loading using canActivate and canResolve. Classes can be provided as well, they should be derived from JssCanActivate and/or JssResolve.
```

Comment:
@jlast Remove extra space in: `a   for`. And I'm not sure, but we don't have `canResolve` inside the package.
So probably, replace it `canResolve` -> `resolve`